src/sentry/static/sentry/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -68,7 +68,27 @@ const LEGACY_BROWSER_SUBFILTERS = {
 
 const LEGACY_BROWSER_KEYS = Object.keys(LEGACY_BROWSER_SUBFILTERS);
 
-class LegacyBrowserFilterRow extends React.Component {
+type FormFieldProps = React.ComponentProps<typeof FormField>;
+
+type RowProps = {
+  data: {
+    active: string[] | boolean;
+  };
+  onToggle: (
+    data: RowProps['data'],
+    filters: RowState['subfilters'],
+    event: React.MouseEvent
+  ) => void;
+  disabled?: boolean;
+};
+
+type RowState = {
```

Comment:
and `State` here?

Reply:
There are two components in this file, a row and container. I used `Props` and `State` for the container which is lower in the file (line 186)